backend/contributions/serializers.py

```
@@ -1020,22 +1043,42 @@ class SubmissionProposeSerializer(serializers.Serializer):
         required=False,
         allow_null=True,
     )
+    rubric_review = serializers.JSONField(required=False)
 
     def validate(self, data):
```

Ruff (0.15.15)

[warning] Too many branches (13 > 12)

(PLR0912)